### PR TITLE
fix(allure): Failed xml parse when assert msg contains ansi symbol

### PR DESCRIPTION
**Pull request by Vorobeyko**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,7 +60,8 @@
     "promise-retry": "^1.1.1",
     "requireg": "^0.1.8",
     "resq": "^1.5.0",
-    "sprintf-js": "^1.1.1"
+    "sprintf-js": "^1.1.1",
+    "strip-ansi": "^5.2.0"
```
Comment (DavertMik) [inline package.json]: can this be replaced with a 1 regex string?

Comment (Vorobeyko) [reply package.json]: I dont know) 
maybe it (https://github.com/chalk/ansi-regex/blob/master/index.js#L4)  will help?

Comment (Vorobeyko) [reply package.json]: I fixed it

**Comment by DavertMik** [comment]
Thanks!

**Pull request merged**